lib/spark.js

```diff
@@ -149,6 +150,16 @@ export default class Spark {
       console.error(err)
     }
 
+    if (stats.timeout) {
+      Zinnia.activity.error('Retrieval timed out')
+    } else if (stats.carTooLarge) {
+      Zinnia.activity.error('CAR too large')
+    } else if (stats.statusCode >= 400) {
+      Zinnia.activity.error('Bad response')
```

Comment:
```suggestion
      Zinnia.activity.error('Retrieval failed')
```

Reply:
The response itself is not necessarily bad. Maybe we are using a protocol that the SP does not support (e.g. bitswap), or there was a timeout on Lassie side while waiting for the next bitswap block, etc.

Reply:
Hmm, you're right that the response itself isn't bad - but it's one where we can't make a retrieval. "Retrieval failed" doesn't tell you anything about what happened however. I would like to find a message that tells us that the SP is not able to serve the retrieval. What about `Unexpected response`?

Reply:
I see. I think you already wrote the message in your text:

```
The Storage Provider is not able to serve the retrieval.
```